src/Dialect/ONNX/DialectBuilder.cpp

```diff
@@ -144,6 +144,22 @@ Value OnnxBuilder::mul(Type resultType, Value A, Value B) const {
   return b().create<ONNXMulOp>(loc(), resultType, toTensor(A), toTensor(B));
 }
 
+Value OnnxBuilder::pad(Type outputType, Value input, Value pads,
+    Value constantValue, std::string mode) const {
+  return b().create<ONNXPadOp>(
+      loc(), outputType, input, pads, constantValue, b().getStringAttr(mode));
+}
+
+Value OnnxBuilder::padZero(
+    Type outputType, Value input, Value pads, std::string mode) const {
+  Type tensorTypeF32 = RankedTensorType::get({}, b().getF32Type());
+  DenseElementsAttr denseAttrZero =
+      DenseElementsAttr::get(tensorTypeF32, ArrayRef<float>({0.0}));
+  Value constantValue = constant(denseAttrZero);
```

Comment:
Does it need to be a tensor of size 0? Can it be just a constant? The standard say
> constant_value (optional, heterogeneous) - T: (Optional) A scalar value to be used if the mode chosen is constant (by default it is 0, empty string or False).

Reply:
I used None value for the `constant_value` since default is 0. Thanks!